### PR TITLE
Feat/34 dropbox nearest location

### PR DESCRIPTION
**Pull request by nickbouldien**
can create a new ticket to find a better png for the "closest dropbox" icon
can create new ticket for the map key for the icons
created https://github.com/Ballot-Drop/ballot-drop/issues/71 for "Find dropbox nearest an entered address"
still not sure how we want to address the "dropbox is in a different county" problem

### CODE REVIEW

File: src/components/DropBoxContainer.vue [hunk 1]
```diff
@@ -29,83 +32,103 @@
         </b-input-group>
       </b-form-group>
 
-      <b-table
-        :items="locations"
-        :fields="fields"
+      <LocationTable
```
Comment (nickbouldien) [inline src/components/DropBoxContainer.vue:35]: extracted this out into a new component

File: src/main.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ Vue.use(IconsPlugin)
 import 'bootstrap/dist/css/bootstrap.css'
 import 'bootstrap-vue/dist/bootstrap-vue.css'
 
-Vue.config.productionTip = false
+Vue.config.productionTip = process.env.NODE_ENV !== "production";
```
Comment (nickbouldien) [inline src/main.js:18]: actually for this, I'd have to see what heroku environment variables we have.  if it's something other than 'production' for NODE_ENV, I might have to change this.  but I don't have access to the heroku project

Comment (pjhoberman) [reply src/main.js:18]: We're not currently setting a var for that. I can create one though

Comment (nickbouldien) [reply src/main.js:18]: can you check if heroku is setting that for us? looks like it's been that way for most node related applications for a while, but I'd like to verify

https://devcenter.heroku.com/changelog-items/688

Comment (pjhoberman) [reply src/main.js:18]: It's not in the config vars, but we can console log it to test

image

**Comment by pjhoberman** [comment]
This is incredible. Great work!

Two things that I think are important to update out as a user:

Can we add something to the infobox on the closest one that signifies it at the closest?
Let's get the marker icon to be as close as possible, just different color. Right now it's slightly off:

image

**Pull request merged**